pydantic_ai_slim/pydantic_ai/output.py

```diff
@@ -60,8 +64,8 @@
 
 TextOutputFunc = TypeAliasType(
     'TextOutputFunc',
-    Callable[[RunContext, str], Awaitable[T_co] | T_co] | Callable[[str], Awaitable[T_co] | T_co],
-    type_params=(T_co,),
+    Callable[[RunContext[TextOutputAgentDepsT], str], Awaitable[T_co] | T_co] | Callable[[str], Awaitable[T_co] | T_co],
```

Comment:
Yeah looking at this now, since this will only work for `TextOutput` and not output functions in general, I don't think it's worth the extra (out of order) type var and would prefer to just have `RunContext[Any]` here and leave it at that